$OpenBSD$

Use unsigned char for color_get instead for int
XXX not tested

--- src/examples/edje-basic.c.orig	Wed Nov  2 08:39:45 2011
+++ src/examples/edje-basic.c	Wed Nov  2 08:40:14 2011
@@ -97,6 +97,7 @@ main(void)
 {
    Evas_Object *border, *bg;
    int x, y, w, h;
+   unsigned char r, g, b, a;
    Evas *evas;
 
    if (!ecore_evas_init())
@@ -180,9 +181,9 @@ main(void)
                    "is: x = %d, y = %d, w = %d, h = %d\n", x, y, w, h);
 
    evas_object_color_get(edje_object_part_object_get(edje_obj, "part_one"),
-                         &x, &y, &w, &h);
+                         &r, &g, &b, &a);
    fprintf(stdout, "That part's color components are: r = %d, g = %d, b = %d,"
-                   " a = %d\n", x, y, w, h);
+                   " a = %d\n", r, g, b, a);
 
    edje_object_size_max_get(edje_obj, &w, &h);
    fprintf(stdout, "The Edje object's max. size is: %d, %d\n", w, h);
